src/main/java/longah/handler/InputHandler.java

```diff
@@ -59,8 +60,8 @@ public static Command parseCommand(String commandString, String taskExpression)
         case "help":
             return new HelpCommand(commandString, taskExpression);
             
-        case "group":
-            // Fallthrough
+        case "switchgroup":
```

Comment:
i thnk can just keep to `group`. Im trying to use human readable commands where possible